README.md

```diff
@@ -56,18 +56,60 @@ See https://getcomposer.org/ for more information and documentation.
 
 ### Inlining Css
 
-This is how to use the `CssInliner` class:
+The most basic way to use the `CssInliner` class is to create an instance with
+the original HTML, inline the external CSS, and then get back the resulting
+HTML:
 
 ```php
-$visualHtml = \Pelago\Emogrifier\CssInliner::fromHtml($html)->inlineCss($css)->render();
+use Pelago\Emogrifier\CssInliner;
+
+…
+
+$visualHtml = CssInliner::fromHtml($html)->inlineCss($css)->render();
+```
+
+If there is no external CSS file and all CSS is located within a `<style>`
+node in the HTML, you can omit the `$css` parameter:
+
+```php
+$visualHtml = CssInliner::fromHtml($html)->inlineCss()->render();
+```
+
+If you would like to get back only the content of the BODY element instead of
+the complete HTML document, you can use the `renderBodyContent` method instead:
+
+```php
+$bodyContent = $visualHtml = CssInliner::fromHtml($html)->inlineCss()
+  ->renderBodyContent();
 ```
 
-You can also use the `DOMDocument` created by `CssInliner` to process it further:
+If you would like to modify the way inlining process with any of the
+available [options](#options), you will need to call the corresponding methods
+before inlining the CSS. The could then would look like this:
```

Comment:
> The could then would look like this:

Should be “The could then look like this:”

Reply:
Done.